### DISCUSSION TITLE
I don't have last name, But it is required in billing information.

### DISCUSSION THREAD

**Discussion by jtuluve** [community/community#74547] [GitHub Education] [answered]
### Hi everyone,

I don't have legal last name. The billing information requires last name. What should I do? Is it okay to use the last name I'm planning to take in future? But it's not in my academic ID or anything and I cannot keep last name blank in billing information.

I want student developer pack. Is it fine to add a random last name and submit the application?

**Comment by developer-aaditya** [comment]
I am also facing this problem..

**Comment by jtuluve** [comment, marked as answer]
I got an email from GitHub, that if I don't have a last name, I should use my first name again as last name. Hope this helps.
